account_move_update_analytic/wizards/account_move_update_analytic.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Le Filament
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMoveUpdateAnalytic(models.TransientModel):
+    _name = "account.move.update.analytic.wizard"
+    _description = "Account Move Update Analytic Account Wizard"
+
+    line_id = fields.Many2one("account.move.line", string="Invoice line")
+    current_analytic_account_id = fields.Many2one(
+        related="line_id.analytic_account_id", string="Current Analytic Account"
+    )
+    current_analytic_tag_ids = fields.Many2many(
+        related="line_id.analytic_tag_ids", string="Current Analytic Tags"
+    )
+    company_id = fields.Many2one(related="line_id.company_id")
+    new_analytic_account_id = fields.Many2one(
+        "account.analytic.account", string="New Analytic Account", check_company=True
+    )
+    new_analytic_tag_ids = fields.Many2many(
+        "account.analytic.tag", string="New Analytic Tags", check_company=True
+    )
+
+    @api.model
+    def default_get(self, fields):
+        rec = super(AccountMoveUpdateAnalytic, self).default_get(fields)
+        context = dict(self._context or {})
+        active_id = context.get("active_id", False)
```

Comment:
You can use `self.env.context` and remove the line above (`context = dict(self._context or {})`)
```suggestion
        active_id = self.env.context.get("active_id")
```